docs/knative-eventing-mesh/01-01-knative-eventing-mesh.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Overview
+---
+
+>**CAUTION:** This implementation will soon replace Kyma Event Bus. Consider it as experimental as it is still under development.
+
+Knative Eventing Mesh allows you to easily integrate external applications with Kyma. Under the hood, Knative Eventing Mesh implements [Knative Eventing](https://knative.dev/docs/eventing/) to ensure Kyma receives business events from external sources and is able to trigger business flows using lambda functions or services. 
+
+Knative Eventing Mesh implementation relies on [Knative Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) custom resources that define the event delivery process. 
```

Comment:
Actually, I mean explaining once that these are Knative elements instead of using first Knative Broker and Broker later on...maybe sth like that?:
`Knative Eventing Mesh implementation relies on [Broker and Trigger](https://knative.dev/docs/eventing/broker-trigger/) custom resources that are Knative components defining the event delivery process.`